mempool/collector.go

```diff
@@ -300,11 +300,15 @@ func ParseTxns(txs []exptypes.MempoolTx, params *chaincfg.Params, lastBlock *Blo
 	revs := make([]exptypes.MempoolTx, 0)
 	regular := make([]exptypes.MempoolTx, 0)
 
-	var totalOut float64
+	var regularTotal, ticketTotal, voteTotal, revTotal float64
+	var likely bool
+
+	var totalOut, likelyTotal float64
 	var totalSize int32
 	votingInfo := exptypes.VotingInfo{
```

Comment:
Seems like a `NewVotingInfo` c'tor would be good to have to ensure setting the max votes correctly and making the maps.  BTW, it could set the initial capacity of the maps to params.TicketsPerBlock, but I have a feeling the initial capacity of a default map is above that anyway.